zerver/views/test_emails.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+from __future__ import absolute_import
+from __future__ import print_function
+import os
+from typing import List, Dict, Any
+from django.http import HttpRequest, HttpResponse
+from django.shortcuts import render
+from django.template import loader, TemplateDoesNotExist
+
+ZULIP_PATH = os.path.join(os.path.dirname(os.path.abspath(__file__)), '../../')
+
+def email_page(request):
+    # type: (HttpRequest) -> HttpResponse
+    # write fake data for all variables
+    test_context = {
+        'user_profile': {
+            'full_name': 'Zooly Zulipson',
+            'email': 'zooly@zulip.org',
+            'realm': {
+                'uri': 'https://chat.zulip.org',
+                'name': 'Zulip Community',
+            },
+        },
+        'realm': {
+            'uri': 'https://chat.zulip.org',
+            'name': 'Zulip Community',
+        },
+        'device_info': {
+            'login_time': "12/12/12, 12:12:12",
+            'device_browser': "Firefox",
+            'device_os': "ZulipOS",
+            'device_ip': '127.0.0.1',
+        },
+        'referrer': {
+            'full_name': 'Zooly Zulipson',
+            'email': 'zooly@zulip.org',
+        },
+        'user': {
+            'full_name': 'Zooly Zulipson',
+            'email': 'zooly@zulip.org',
+        },
+        'referrer_name': 'Zooly Zulipson',
+        'referrer_email': 'Zooly Zulipson',
+        'realm_uri': 'https://chat.zulip.org',
+        'server_uri': 'https://chat.zulip.org',
+        'old_email': 'oldmail@zulip.org',
+        'new_email': 'newmail@zulip.org',
+        'activate_url': 'https://test.zulip.org',
+        'support_email': 'admin@zulip.org',
+        'zulip_support': 'admin@zulip.org',
+        'unsubscribe_link': 'https://unsub.zulip.org',
+    }
```

Comment:
That function only generated 3 of the above fields. Do you think it's still worth including?